neo/Persistence/IReadOnlyStore.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace Neo.Persistence
+{
+    public interface IReadOnlyStore
+    {
```

Comment:
We can change the `table` to a `enum`?
We can have multiple entries in this enum for plugins that want to store information, but for me is more clear with enums

Reply:
The low level `IStore` doesn't need to know these enum names. And `StoreView` has used ·DataCache· to hide this information, so no enum is needed.